diff --git a/build/ios/Swift.d.ts b/build/ios/Swift.d.ts
index 1501a70d4a22203421e71964b7973266bc65ede9..e66aac1cca72b0e3536d5143b037173eca9e8473 100644
--- a/build/ios/Swift.d.ts
+++ b/build/ios/Swift.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="xcode" />
 import { ConfigPlugin, XcodeProject } from '../Plugin.types';
 /**
  * Ensure a Swift bridging header is created for the project.
diff --git a/build/ios/XcodeProjectFile.d.ts b/build/ios/XcodeProjectFile.d.ts
index 6c23a505762d0b7ba0f7cb5a382f7883e46345d3..730336c26b771771ca54c05dd2c8a7fba75fbea5 100644
--- a/build/ios/XcodeProjectFile.d.ts
+++ b/build/ios/XcodeProjectFile.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="xcode" />
 import { ConfigPlugin, XcodeProject } from '../Plugin.types';
 /**
  * Create a build source file and link it to Xcode.